nats-concepts/jetstream/consumers.md

```diff
@@ -1,234 +1,91 @@
+| Field | Description | Version | Editable |
+| :--- | :--- | :--- | :--- |
+| Durable | If set, clients can have subscriptions bind to the consumer and _resume_ until the consumer is explicitly deleted. A durable name cannot contain whitespace, `.`, `*`, `>`, path separators (forward or backwards slash), and non-printable characters. | 2.2.0 | No |
+| FilterSubject | An overlapping subject with the subjects bound to the stream which will filter the set of messages received by the consumer. | 2.2.0 | Yes |
+| [AckPolicy](#ackpolicy) | The requirement of client acknowledgements, either `AckExplicit`, `AckNone`, or `AckAll`. | 2.2.0 | No |
+| AckWait | The duration that the server will wait for an ack for any individual message _once it has been delivered to a consumer_. If an ack is not received in time, the message will be redelivered. | 2.2.0 | Yes |
+| [DeliverPolicy](#deliverpolicy) | The point in the stream to receive messages from, either `DeliverAll`, `DeliverLast`, `DeliverNew`, `DeliverByStartSequence`, `DeliverByStartTime`, or `DeliverLastPerSubject`. | 2.2.0 | No |
+| OptStartSeq | Used with the `DeliverByStartSequence` [deliver policy](#deliverpolicy).  | 2.2.0 | No |
+| OptStartTime | Used with the `DeliverByStartTime` [deliver policy](#deliverpolicy). | 2.2.0 | No |
+| Description | A description of the consumer. This can be particularly useful for ephemeral consumers to indicate their purpose since the durable name cannot be provided. | 2.3.3 | Yes |
+| InactiveThreshold | Duration that instructs the server to cleanup consumers that are inactive for that long. Prior to 2.9, this only applied to ephemeral consumers. | 2.2.0 | Yes |
+| [MaxAckPending](#maxackpending) | Defines the maximum number of messages, without an acknowledgement, that can be outstanding. Once this limit is reached message delivery will be suspended. This limit applies across _all_ of the consumer's bound subscriptions. A value of -1 means there can be any number of pending acks (i.e. no flow control). This does not apply when the `AckNone` policy is used. | 2.2.0 | Yes |
+| MaxDeliver | The maximum number of times a specific message delivery will be attempted. Applies to any message that is re-sent due to ack policy (i.e. due to a negative ack, or no ack sent by the client). | 2.2.0 | Yes |
+| ReplayPolicy | If the policy is `ReplayOriginal`, the messages in the stream will be pushed to the client at the same rate that they were originally received, simulating the original timing of messages. If the policy is `ReplayInstant` \(the default\), the messages will be pushed to the client as fast as possible while adhering to the Ack Policy, Max Ack Pending and the client's ability to consume those messages. | 2.2.0 | No |
+| Replicas | Sets the number of replicas for the consumer's state. By default, consumers inherit the number of replicas from the stream. | 2.8.3 | Yes |
```

Comment:
About consumer number of replicas maybe expand to say that the default value 0 means it inherits the number from the stream's